<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialTerminal</class>
 <widget class="QMainWindow" name="serialTerminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Terminal </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="txtRecieved">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>961</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>Text recieved from Microcontroller</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtToSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>871</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Text to send to Microcontroller</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>send.svg</normaloff>send.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbSerialPorts">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>0</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="currentText">
     <string extracomment="Please Select"/>
    </property>
   </widget>
   <widget class="QLabel" name="labSerialPortStatusDisconnected">
    <property name="geometry">
     <rect>
      <x>798</x>
      <y>4</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>power_off.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Serial Terminal </string>
    </property>
   </widget>
   <widget class="QLabel" name="labSerialPortStatusConected">
    <property name="geometry">
     <rect>
      <x>798</x>
      <y>4</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>electrical_services.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
